src/sidebar/util/test/time-test.js

```diff
@@ -5,128 +5,166 @@ const time = require('../time');
+      { now: '1970-01-01T04:00:00.000Z', text: '4 hrs' },
+    ].forEach(test => {
+      it('creates correct fuzzy string for fixture ' + test.now, () => {
+        const timeStamp = fakeDate('1970-01-01T00:00:00.000Z');
+        const now = fakeDate(test.now);
+        assert.equal(time.toFuzzyString(timeStamp, now), test.text);
+      });
+    });
 
-    for (let i = 0, f; i < FIXTURES_TO_FUZZY_STRING.length; i++) {
-      f = FIXTURES_TO_FUZZY_STRING[i];
-      it('creates correct fuzzy string for fixture ' + i, testFixture(f));
-    }
+    [
+      {
+        now: '1970-01-02T03:00:00.000Z',
+        text: '2 Jan',
+        options: { day: 'numeric', month: 'short' },
+      },
+      {
+        now: '1970-01-04T00:30:00.000Z',
+        text: '4 Jan',
+        options: { day: 'numeric', month: 'short' },
+      },
+      {
+        now: '1970-07-03T00:00:00.000Z',
+        text: '3 July',
+        options: { day: 'numeric', month: 'short' },
+      },
+      {
+        now: '1971-01-01T00:00:00.000Z',
+        text: '1 Jan 1970',
+        options: { day: 'numeric', month: 'short', year: 'numeric' },
+      },
+      {
+        now: '1971-03-01T00:00:00.000Z',
+        text: '1 Jan 1970',
+        options: { day: 'numeric', month: 'short', year: 'numeric' },
+      },
+      {
+        now: '1972-01-01T00:00:00.000Z',
+        text: '1 Jan 1970',
+        options: { day: 'numeric', month: 'short', year: 'numeric' },
+      },
+      {
+        now: '1978-01-01T00:00:00.000Z',
+        text: '1 Jan 1970',
+        options: { day: 'numeric', month: 'short', year: 'numeric' },
+      },
+    ].forEach(test => {
+      it(
+        'passes correct arguments to format fuzzy string date for fixture ' +
+          test.now,
+        () => {
+          const timeStamp = fakeDate('1970-01-01T00:00:00.000Z');
+          const now = fakeDate(test.now);
+
+          fakeIntl.DateTimeFormat().format.returns(test.text); // eslint-disable-line new-cap
+          assert.equal(time.toFuzzyString(timeStamp, now, fakeIntl), test.text);
+          assert.calledWith(fakeIntl.DateTimeFormat, undefined, test.options);
+          assert.calledWith(fakeIntl.DateTimeFormat().format, timeStamp); // eslint-disable-line new-cap
+        }
+      );
+    });
 
     it('falls back to simple strings for >24hrs ago', function() {
       // If window.Intl is not available then the date formatting for dates
       // more than one day ago falls back to a simple date string.
-      const d = new Date().toISOString();
-      sandbox.clock.tick(day * 2 * 1000);
+      const timeStamp = fakeDate('1970-01-01T00:00:00.000Z');
+      timeStamp.toDateString = sinon.stub().returns('Thu Jan 01 1970');
+      const now = fakeDate('1970-01-02T00:00:00.000Z');
 
-      assert.equal(time.toFuzzyString(d, null), 'Thu Jan 01 1970');
+      assert.equal(time.toFuzzyString(timeStamp, now, null), 'Thu Jan 01 1970');
+      assert.calledOnce(timeStamp.toDateString);
```

Comment:
You probably want to reverse the order of these asserts since the first one will lead to the second one.